modules/light-clients/06-solomachine/client_state.go

```diff
@@ -43,7 +43,7 @@ func (cs ClientState) GetLatestHeight() exported.Height {
 
 // GetTimestampAtHeight returns the timestamp in nanoseconds of the consensus state at the given height.
 func (cs ClientState) GetTimestampAtHeight(
-	_ sdk.Context,
+	ctx sdk.Context,
```

Comment:
nit: consistency.

Reply:
love it.  finishing up with the merge, then will finish the rest here.

Reply:
Jim, I've now disabled this linter and I think that things are better off named to reduce dev time to search for the things to be named.  In cases where we've got an interface where we never use a prameter, then I figure we just remove the parameter.